app/screens/post_options/post_option.js

```diff
@@ -20,6 +20,7 @@ import flag from 'assets/images/post_menu/flag.png';
 import link from 'assets/images/post_menu/link.png';
 import pin from 'assets/images/post_menu/pin.png';
 import trash from 'assets/images/post_menu/trash.png';
+import reply from 'assets/images/post_menu/reply.png';
```

Comment:
perhaps we should remove this component `app/components/reply_icon` as we now have assets for it, and lets take into account that the current reply_icon component is an svg which is not as performant as an asset.

thoughts?